### PR TITLE
Add guides/request-validation-php-lumen examples 1, 2 and 3

### PR DESCRIPTION
**Pull request by joliveros**

### CODE REVIEW

File: guides/request-validation-php-lumen/example-1/example-1.5.x.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      $isValid = $requestValidator->validate(
+        $request->header('X-Twilio-Signature'),
+        $request->fullUrl(),
+        $request->toArray()
+      );
+
+      if ($isValid) {
+        return $next($request);
+      } else {
+        return (new Response('You are not Twilio :(', 403));
```
Comment (acamino) [inline guides/request-validation-php-lumen/example-1/example-1.5.x.php]: Can you clarify why are you using the enclosing parens?

File: guides/request-validation-php-lumen/example-2/example-2.5.x.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Http\Request;
+use Twilio\Twiml;
+
+$app->post('/voice', ['middleware' => 'TwilioRequestValidator',
+  function () use ($app) {
```
Comment (acamino) [inline guides/request-validation-php-lumen/example-2/example-2.5.x.php]: Not sure why you're the `use` identifier to pass `$app` into the scope of this anonymous function. I can't see this function using `$app`. Could you provide me some context on this please?

Comment (acamino) [inline guides/request-validation-php-lumen/example-2/example-2.5.x.php]: Additionally the preferred style for working with PHP functions is:

```php
function() {
  //    ^ notice there is no space between the `function` identifier
  //      and the open paren.
  // ...
}

File: guides/request-validation-php-lumen/example-2/example-2.5.x.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Http\Request;
+use Twilio\Twiml;
+
+$app->post('/voice', ['middleware' => 'TwilioRequestValidator',
```
Comment (acamino) [inline guides/request-validation-php-lumen/example-2/example-2.5.x.php]: Could you double check if it is needed to prepend a `/` to the route name please?

File: guides/request-validation-php-lumen/example-2/example-2.5.x.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+$app->post('/message', ['middleware' => 'TwilioRequestValidator',
+  function (Request $request) use ($app) {
+    $bodyLength = strlen($request->input('Body'));
+
+    $twiml = new Twiml();
+    $twiml->message("Your text to me was $bodyLength characters long. ".
+                    "Webhooks are neat :)");
+
+    return response($twiml)
```
Comment (acamino) [inline guides/request-validation-php-lumen/example-2/example-2.5.x.php]: Could be this a one-liner?

File: guides/request-validation-php-lumen/example-3/example-3.5.x.php [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    {
+      if (env('APP_ENV') === 'test') {
+        return $next($request);
+      }
+
+      // Be sure TWILIO_APP_TOKEN is set in your .env file.
+      // You can get your app token in your twilio console https://www.twilio.com/console
+      $requestValidator = new RequestValidator(env('TWILIO_APP_TOKEN'));
+
+      $isValid = $requestValidator->validate(
```
Comment (acamino) [inline guides/request-validation-php-lumen/example-3/example-3.5.x.php]: This looks nice!

File: guides/request-validation-php-lumen/example-1/example-1.5.x.php (hunk 1 above)
Comment (joliveros) [inline guides/request-validation-php-lumen/example-1/example-1.5.x.php]: smh

File: guides/request-validation-php-lumen/example-2/example-2.5.x.php (hunk 3 above)
Comment (joliveros) [inline guides/request-validation-php-lumen/example-2/example-2.5.x.php]: doesn't looks like it is needed. I modified the code and ran it and it works perfectly without it. thanks!

File: guides/request-validation-php-lumen/example-2/example-2.5.x.php (hunk 2 above)
Comment (joliveros) [inline guides/request-validation-php-lumen/example-2/example-2.5.x.php]: You're absolutely right `$app` is not needed there.

**Comment by acamino** [comment]
Great work! **LGTM**

**Pull request merged**